src/models/AppState.js

```diff
@@ -55,6 +55,21 @@ function (Okta, Q, Factor, BrowserFeatures, Errors) {
     });
   }
 
+  function getMinutesString(factorLifetimeInMinutes) {
+    if (factorLifetimeInMinutes > 60 && factorLifetimeInMinutes <= 1440) {
+      var lifetimeInHours = (factorLifetimeInMinutes / 60);
+      return Okta.loc('hours', 'login', [lifetimeInHours]);
+    } else if (factorLifetimeInMinutes > 1440 && factorLifetimeInMinutes < 2880) {
+      var lifetimeInDays = (factorLifetimeInMinutes / 1440);
+      return Okta.loc('days', 'login', [lifetimeInDays]);
+    }
```

Comment:
`60 < factorLifetimeInMinutes <=1440` and `1440 < factorLifetimeInMinutes < 2880` ?